src/commands/misc/dog.js

```diff
@@ -11,9 +11,18 @@ module.exports = class Dog extends Command {
   run (message) {
     message.channel.startTyping()
     request.get('https://random.dog/woof.json').then(r => {
-      const embed = this.client.getDefaultEmbed(message.author)
-      embed.setImage(r.body.url).setDescription('Here is your dog <:DoggoF:445701839564963840>')
-      message.channel.send({embed})
+      var notsupported = ['.mp4']
+      if (!r.body.url.endsWith(notsupported)) {
+        const embed = this.client.getDefaultEmbed(message.author)
+        embed.setImage(r.body.url).setDescription('Here is your dog <:DoggoF:445701839564963840>')
+        message.channel.send({embed})
+      } else {
+        request.get('https://random.dog/woof.json').then(r => {

```

Comment:
I belive a recursive function checking if the file aren't a mp4 is better here, it totally can gets another .mp4 after the first one

Reply:
i can fix this with a thing